benchexec/tools/veriabs.py

```diff
@@ -25,8 +25,6 @@ class Tool(benchexec.tools.template.BaseTool2):
         "afl-2.35b",
         "verifuzz",
         "afl-2.35b_v1",
-        "frama-c-Chlorine-20180502",
-        "UAutomizer-linux",
```

Comment:
I would say this should be kept because otherwise the tool-info module will not work correctly anymore for older versions of VeriAbs. If these paths do not exist in the current version of VeriAbs this is not a problem because non-matching paths will simply be ignored.

Reply:
Thanks for the input @PhilippWendler ! , I will add the paths in the required_paths section.